CONTRIBUTING.md

```diff
@@ -651,3 +651,79 @@ Names of files and directories should be in lowercase, with dashes between words
 
   As an exception, an explicit conditional expression with null can be used when fixing a important bug without triggering a mass rebuild.
   If this is done a follow up pull request _should_ be created to change the code to `lib.optional(s)`.
+
+## I opened a PR, how do I get it merged?
+[i-opened-a-pr-how-do-i-get-it-merged]:#i-opened-a-pr-how-do-i-get-it-merged
+
+In order for your PR to be merged, someone with merge permissions on the repository needs to review and merge it.
+Because the group of people with merge permissions is mostly a collection of independent unpaid volunteers who do this in their own free time, this can take some time to happen. It is entirely normal for your PR to sit around without any feedback for days, weeks or sometimes even months. We strive to avoid the latter cases of course but the reality of it is that this does happen quite frequently. Even when you get feedback, further feedback may take similarly long. Don't be intimidated by this and kindly ask for feedback again every so often. If your change is good it will eventually be merged at some point.
+
+There are some things you can do to help speed up the process of your PR being merged though. In order to speed the process up, you need to know what needs to happen before a committer will actually hit the merge button. This section intends to give a little overview and insight of what happens after you create your PR.
```

Comment:
You use the word committer here without defining it before. I think this might be a bit confusing. Maybe write "the group of people with merge permissions (the committers)" in the paragraph before.